actions/service.go

```diff
@@ -27,95 +27,71 @@ type Task struct {
 }
 
 type RunResult struct {
-	RunID      string
-	BranchID   string
-	OnRef      string
```

Comment:
Why removed?

Reply:
Added as SourceRef - let me know if I missed something else, there was no other OnRef on the original event.